### PR TITLE
PODB-266 Add the ability to set debugging mode

### PR DESCRIPTION
**Pull request by zlayaAvocado**
## Summary of Changes

Adds the ability to set debugging mode, which is `false` by default.

## Testing

Checked that I am able to set the debugging true / false mode with the function.

- [ ] I have added automated tests for my changes

### CODE REVIEW

File: src/LtiServiceConnector.php [hunk 1]
```diff
@@ -108,14 +114,16 @@ public function makeServiceRequest(
...
+        if ($this->debuggingMode) {
+            error_log('Syncing grade for this lti_user_id: '.
+                json_decode($request->getPayload()['body'])->userId.' '.print_r([
+                    'request_method' => $request->getMethod(),
+                    'request_url' => $request->getUrl(),
+                    'request_body' => $request->getPayload()['body'],
+                    'response_headers' => $respHeaders,
+                    'response_body' => (string) $respBody,
+                ], true));
+        }
```
[the remark this answers is not in the document]
Comment (zlayaAvocado) [reply src/LtiServiceConnector.php:126]: good point, I will update tha

**Comment by dbhynds** [comment]
@zlayaAvocado and @Watercycle Just a note for future reference, too. We should update `UPGRADES.md` whenever we make breaking changes. I'll open a PR for that now.

**Comment by zlayaAvocado** [comment]
@dbhynds why is it a breaking change? nothing broke unless I am unaware of something

**Comment by Watercycle** [comment]
Also wondering this. Removing recently added debugging logs and adding a new method seems like a minor version bump.

**Pull request merged**